src/app/share/types/index.ts

```diff
@@ -0,0 +1,47 @@
+import { SharedFiles, SharedFolders } from '@internxt/sdk/dist/drive/share/types';
+
+export type AdvancedSharedLink = SharedFolders &
+  SharedFiles & {
+    isFolder: boolean;
+    isRootLink: boolean;
+  };
+
+export type OrderBy = { field: 'views' | 'createdAt'; direction: 'ASC' | 'DESC' } | undefined;
+
+export type SharedLinkItemType = {
+  id: string;
+  folderId: string;
+  ownerId: string;
+  sharedWith: string;
+  encryptionKey: string;
+  createdAt: string;
+  updatedAt: string;
+  owner_id: string;
```

Comment:
Try to avoid using this fields, if they arrive, as we should just use camelCase

Reply:
hey this entire type was created by Ramon but I think it is not ended and it is not being used as it was planned. I've decided to remove it, so when Ramon comes back we will discuss it to see if it is really necessary or not. We can always bring it back in that case